cli/sub_command.go

```diff
@@ -73,6 +89,10 @@ func configureSubCommand(app commandHost) {
 	act.Flag("raw", "Show the raw data received").Short('r').UnNegatableBoolVar(&c.raw)
 	act.Flag("translate", "Translate the message data by running it through the given command before output").StringVar(&c.translate)
 	act.Flag("ack", "Acknowledge JetStream message that have the correct metadata").BoolVar(&c.jsAck)
+	//We do not support (explicit) ackPolicy right now. The only situation where it is useful would be WorkQueue policy right now.
```

Comment:
Please add a space between `//` and the text that's go idiomatic, ones without the space is kind of reserved for some kind of compiler directives